src/TextArea/ErrorMessage.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+import { Typography } from '../Typography'
+
+export const ErrorMessage = ({ error, preserveSpace }) => {
+  if (error) {
+    return (
+      <Typography variant='textSmall' color='error'>
+        {error}
+      </Typography>
+    )
+  }
+
+  if (preserveSpace) {
+    return <Dummy />
+  }
+
+  return null
+}
+
+const Dummy = styled.div`
+  display: ${({ hide }) => (hide ? 'none' : 'block')};
+  height: 20px;
+  opacity: 0;
+  visibility: hidden;
```

Comment:
Either use:

```
  return <Dummy hide={!preserveSpace} />
}

const Dummy = styled.div`
  display: ${({ hide }) => (hide ? 'none' : 'block')};
  height: 20px;
  opacity: 0;
  visibility: hidden;
```

or

```suggestion
  if (preserveSpace) {
    return <Dummy />
  }

  return null
}

const Dummy = styled.div`
  display: block;
  height: 20px;
  opacity: 0;
  visibility: hidden;
```